src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,33 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'grasp')
+def fuze_grasp(robot, fuze, manip=None):
```

Comment:
Okay I think we have this addressed properly now. The PushGrasp action takes a push_distance parameter and passes that onto the TSRlibrary which can use it or ignore it as appropriate.